training/performance/README.md

```diff
@@ -118,8 +131,25 @@ For example [Megatron-LM](https://github.com/NVIDIA/Megatron-LM) published the f
 | 530B  | 56.0% | 57.0% |
 | 1T    | 56.3% | 57.0% |
 
+As you can see, since Megatron-LM was using activation recomputation for these trainings, MFU < HFU.
+
 More recent H100+A100 MFU/HFU numbers have been published [here](https://github.com/mosaicml/llm-foundry/tree/main/scripts/train/benchmarking#mfu-and-hfu).
 
+Now, whenever you see MFU or HFU numbers published you have to be careful comparing those numbers to any other similar numbers, until you know that the same way was used to calculate FLOPS. Since `HFU=Actual_Achieved_FLOPS/Theoretical_FLOPS` and `Theoretical_FLOPS` are fixed. The only variable here is the `
+Actual_Achieved_FLOPS` and since most of the time an estimated value is calculated based on parameter shapes, there are many versions of calculations out there, some of which are slightly to very imprecise. Moreover you don't know how iteration time was measured.
```

Comment:
One more thing to note is that optimization passes applied by compilers to the training model could also impact the actual achieved value. Often these compilers are not transparent about what they did. Assuming that these passes reduced the number of operations performed reality, the MFU will be optimistic.

On the other hand if the compiler did activation checkpointing silently, the opposite will be true.

TLDR is that you cannot be sure when compilers are in play. What's important is that all of us engineers communicate what we used to measure knowing that it might be flawed.

Reply:
That's important - thank you, Carlos.

I added your note about compilers. If further improvements can be made please suggest.